### PR TITLE
Improved coverage

### PR DESCRIPTION
**Pull request by heshvab**
Improved coverage for some of functions, hooks and components before demo. Those are tests for some of the logic related to cart.

### CODE REVIEW

File: code/src/utils/set-local-storage-cart/setLocalStorageCart.test.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import { LOCAL_STORAGE_KEYS } from "@/constants/common";
+
+import setLocalStorageCart from "./setLocalStorageCart";
```
Comment (D1nviz) [inline code/src/utils/set-local-storage-cart/setLocalStorageCart.test.ts]: use alias

File: code/src/utils/get-local-storage-cart/getLocalStorageCart.test.ts [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import { LOCAL_STORAGE_KEYS } from "@/constants/common";
+
+import getLocalStorageCart from "./getLocalStorageCart";
```
Comment (D1nviz) [inline code/src/utils/get-local-storage-cart/getLocalStorageCart.test.ts]: Also alias

File: code/src/hooks/use-remove-from-cart/useRemoveFromCart.test.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
+import { renderHook } from "@testing-library/react";
+
+import { useAppDispatch } from "@/hooks/use-redux/useRedux";
+import { useRemoveFromCartMutation } from "@/store/api/cartApi";
+import { useUserDetailsSelector } from "@/store/slices/userSlice";
+import { CartItem } from "@/types/cart.types";
+import { UserDetails } from "@/types/user.types";
+
+import useRemoveFromCart from "./useRemoveFromCart";
```
Comment (D1nviz) [inline code/src/hooks/use-remove-from-cart/useRemoveFromCart.test.ts]: Use alias

File: code/src/hooks/use-add-to-cart/useAddToCart.test.ts [hunk 4]
```diff
@@ -0,0 +1,69 @@
+import { renderHook } from "@testing-library/react";
+
+import { useAppDispatch } from "@/hooks/use-redux/useRedux";
+import { useAddToCartMutation } from "@/store/api/cartApi";
+import { useUserDetailsSelector } from "@/store/slices/userSlice";
+import { CartItem } from "@/types/cart.types";
+import { UserDetails } from "@/types/user.types";
+
+import useAddToCart from "./useAddToCart";
```
Comment (D1nviz) [inline code/src/hooks/use-add-to-cart/useAddToCart.test.ts]: Alias

File: code/src/containers/cart-drawer/cart-drawer-item/CartDrawerItem.test.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+
+import CartDrawerItem from "./CartDrawerItem";
```
Comment (D1nviz) [inline code/src/containers/cart-drawer/cart-drawer-item/CartDrawerItem.test.tsx]: Alias

File: code/src/containers/cart-drawer/CartDrawer.test.tsx [hunk 6]
```diff
@@ -0,0 +1,142 @@
+import { fireEvent, render, screen } from "@testing-library/react";
+import { useNavigate } from "react-router-dom";
+
+import { useDrawerContext } from "@/context/drawer/DrawerContext";
+import { useModalContext } from "@/context/modal/ModalContext";
+import useCartItems from "@/hooks/use-cart-items/useUserCartItems";
+import { UserDetails } from "@/types/user.types";
+import formatPrice from "@/utils/format-price/formatPrice";
+
+import CartDrawer from "./CartDrawer";
```
Comment (D1nviz) [inline code/src/containers/cart-drawer/CartDrawer.test.tsx]: Alias

File: code/src/utils/set-local-storage-cart/setLocalStorageCart.test.ts (hunk 1 above)
Comment (heorhiiss) [reply code/src/utils/set-local-storage-cart/setLocalStorageCart.test.ts]: Thank you, Fixed

File: code/src/utils/get-local-storage-cart/getLocalStorageCart.test.ts (hunk 2 above)
Comment (heorhiiss) [reply code/src/utils/get-local-storage-cart/getLocalStorageCart.test.ts]: Thank you, Fixed

File: code/src/hooks/use-remove-from-cart/useRemoveFromCart.test.ts (hunk 3 above)
Comment (heorhiiss) [reply code/src/hooks/use-remove-from-cart/useRemoveFromCart.test.ts]: Thank you, Fixed

File: code/src/hooks/use-add-to-cart/useAddToCart.test.ts (hunk 4 above)
Comment (heorhiiss) [reply code/src/hooks/use-add-to-cart/useAddToCart.test.ts]: Thank you, Fixed

File: code/src/containers/cart-drawer/cart-drawer-item/CartDrawerItem.test.tsx (hunk 5 above)
Comment (heorhiiss) [reply code/src/containers/cart-drawer/cart-drawer-item/CartDrawerItem.test.tsx]: Thank you, Fixed

File: code/src/containers/cart-drawer/CartDrawer.test.tsx (hunk 6 above)
Comment (heorhiiss) [reply code/src/containers/cart-drawer/CartDrawer.test.tsx]: Thank you, Fixed

**Review by OliaHrytsak — APPROVED** [review]

**Review by D1nviz — APPROVED** [review]

**Review by dmtrth25 — APPROVED** [review]

**Review by mhevyk — APPROVED** [review]

**Pull request merged**